functions/packages.sh

```diff
@@ -605,3 +605,63 @@ nginx_setup() {
+  FAILED=0
+  introtext="This will install tellstick core services to enable support for usb connected Tellstick and Tellstick duo. For more details, have a look at http://developer.telldus.se/"
+  failtext="Sadly there was a problem setting up tellstick core. Please report this problem in the openHAB community forum or as a openHABian GitHub issue."
+  successtext="Success, please reboot your system to complete the installation.
+
+Next, add your devices in /etc/tellstick.conf. 
+To detect device id:s use commanline tool: tdtool-improved --event
+When devices is added restart telldusd.service by using: sudo systemctl restart telldusd
+or just reboot the system.
+"
+  if [ -n "$INTERACTIVE" ]; then
+    if ! (whiptail --title "Description, Continue?" --yes-button "Continue" --no-button "Back" --yesno "$introtext" 15 80) then echo "CANCELED"; return 0; fi
+  fi
+  echo -n "$(timestamp) [openHABian] Installing tellstick-core... "
+
+  # Maybe add new repository to be able to install libconfuse1
+  if is_xenial ; then
+    echo 'APT::Default-Release "xenial";' > /etc/apt/apt.conf.d/01release
+    echo "deb http://archive.ubuntu.com/ubuntu/ artful universe" > /etc/apt/sources.list.d/ubuntu-artful.list
+    cond_redirect apt update
+    cond_redirect apt -y -t=artful install libconfuse1
+  fi
+  if is_jessie ; then
+    echo 'APT::Default-Release "jessie";' > /etc/apt/apt.conf.d/01release
+    echo "deb http://deb.debian.org/debian stretch main" > /etc/apt/sources.list.d/debian-stretch.list
+    cond_redirect apt update
+    cond_redirect apt -y -t=stretch install libconfuse1
+  fi
+
+  cond_redirect wget -O - https://s3.eu-central-1.amazonaws.com/download.telldus.com/debian/telldus-public.key | apt-key add -
+  echo "deb https://s3.eu-central-1.amazonaws.com/download.telldus.com unstable main" > /etc/apt/sources.list.d/telldus-unstable.list
+  cond_redirect apt update
+  cond_redirect apt -y install libjna-java telldus-core || FAILED=1
+  if [ $FAILED -eq 0 ]; then echo "OK"; else echo "FAILED"; fi
+
+  echo -n "$(timestamp) [openHABian] Setting up systemd service for telldusd... "
+  cond_redirect cp ${BASEDIR}/includes/tellstick-core/telldusd.service /lib/systemd/system/telldusd.service
+  cond_redirect systemctl daemon-reload || FAILED=1
+  cond_redirect systemctl enable telldusd || FAILED=1
+  cond_redirect systemctl start telldusd.service || FAILED=1
+  if [ $FAILED -eq 0 ]; then echo "OK"; else echo "FAILED"; fi
+
+  echo -n "$(timestamp) [openHABian] Setting up tdtool-improved... "
+  cond_redirect chmod +x ${BASEDIR}/includes/tellstick-core/tdtool-improved.py
+  cond_redirect ln -sf ${BASEDIR}/includes/tellstick-core/tdtool-improved.py /usr/bin/tdtool-improved
+  echo "OK"
+
+  echo -n "$(timestamp) [openHABian] Trying to add Tellstick binding to openHAB... "
+  cond_redirect curl -X POST --header "Content-Type: application/json" --header "Accept: application/json" "http://localhost:8080/rest/extensions/binding-tellstick/install"
```

Comment:
This step is interesting. It would be the first time the openHABian Configuration Tool actually changes settings in openHAB.
I am not against the idea but as it is new, we should discuss it in an independent issue/PR. My first suggestion would for example to define a function `openhab_install_binding`, which also checks the availability of openHAB and waits till the binding was indeed successfully installed.

Would you be okay to leave out those two lines for now, and instead work on the broader idea?